Project/zepeto-modules/Assets/Zepeto Gesture Module/ZepetoScript/GestureLoader.ts

```diff
@@ -1,31 +1,36 @@
 import { ZepetoScriptBehaviour } from 'ZEPETO.Script';
-import { LocalPlayer, SpawnInfo, ZepetoCharacter, ZepetoPlayers } from 'ZEPETO.Character.Controller';
-import { OfficialContentType, WorldService, ZepetoWorldContent, Content } from 'ZEPETO.World';
-import { RawImage, Text, Button } from 'UnityEngine.UI';
-import { GameObject, Texture2D, Transform, WaitUntil } from 'UnityEngine';
+import { LocalPlayer, CharacterState, ZepetoCharacter, ZepetoPlayers} from 'ZEPETO.Character.Controller';
+import { OfficialContentType, ZepetoWorldContent, Content } from 'ZEPETO.World';
+import { Button } from 'UnityEngine.UI';
+import { Object, GameObject, Transform, AnimationClip, WaitForSeconds, Coroutine } from 'UnityEngine';
 import Thumbnail from './Thumbnail';
 
 export default class GestureLoader extends ZepetoScriptBehaviour {
 
     @HideInInspector() public contents: Content[] = [];
     @HideInInspector() public thumbnails: GameObject[] = [];
+    @HideInInspector() public gestureLoop: Coroutine;
+    @HideInInspector() public animation: AnimationClip = null;
 
     @SerializeField() private _loadContentsCount: number = 100;
     @SerializeField() private _contentsParent: Transform;
     @SerializeField() private _prefThumb: GameObject;
 
-    private _myCharacter: ZepetoCharacter;
+    private _myCharacter: ZepetoCharacter ;
     
+    // Loop setting
+    @Header("Looping Setting") 
+    @Tooltip("Activate/Deactivate the looping feature") public isLooping: boolean;
```

Comment:
The `isLooping` flag is applied only to the 'gesture' as introduced in this PR. (not 'pose'). So I suggest clarifying this in the tooltip, document, plus anywhere you find needed. Changing the variable name to `isGestureLooping` MIGHT help a better understanding.

Reply:
Yeah, will update this.